src/components/HomeCards/HomeCards.component.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {
+  Card,
+  CardLink,
+  CardHeader,
+  CardBody,
+  CardTitle,
+  CardText,
+} from './HomeCards.styles';
+
+function HomeCards({ videoId, imgUrl, title, description }) {
```

Comment:
This component is for only one Card, so I think the real name for this file is `HomeCard.component.jsx`